homework/index.js

```diff
@@ -28,14 +28,42 @@
     });
     return elem;
   }
+  function createTable(table, header, value) {
```

Comment:
You are not creating a table here, you are creating or adding a table row. So perhaps `addRow()` would be a better name?